schema/archive.json

```diff
@@ -0,0 +1,210 @@
+{
+  "collections": {
+    "archives": {
+      "name": "archives",
+      "config": {
+        "displayFields": [
+          "id", "target.product", "target.channel", "target.version",
+          "target.platform", "target.locale"
+        ],
+        "schema": {
+          "title": "Archives",
+          "description": "Mozilla software builds.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "id",
+            "source",
+            "download",
+            "target"
+          ],
+          "properties": {
+            "id": {
+              "type": "string",
+              "title": "ID",
+              "description": "Build Hub ID"
+            },
+            "schema": {
+              "type": "integer",
+              "title": "Schema version",
+              "description": "Schema version"
+            },
+            "build": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "build_id": {
+                  "type": "string",
+                  "title": "Build ID",
+                  "description": "Build ID"
+                },
+                "build_date": {
+                  "type": "string",
+                  "title": "Build date",
+                  "description": "i.e: 2017-05-11",
+                  "pattern": "^(19|2[0-5])[0-9]{2}-((0[0-9])|(1[0-2]))-(([0-2][0-9])|(3[0-1]))$"
+                },
+                "sunset_date": {
+                  "type": "string",
+                  "title": "Sunset date",
+                  "description": "i.e: 2017-08-24",
+                  "pattern": "^(19|2[0-5])[0-9]{2}-((0[0-9])|(1[0-2]))-(([0-2][0-9])|(3[0-1]))$"
+                },
+                "build_type": {
+                  "type": "string",
+                  "title": "Build type",
+                  "description": "Build type",
+                  "enum": [
+                    "nightly",
+                    "beta",
+                    "aurora",
+                    "release",
```

Comment:
What does opt means? If you look at the table that Will sent us, you will see that they are using the one from the enum list.